### PR TITLE
[docs] link default property to style property getter/setter docs.

### PR DESCRIPTION
**Pull request by ank27**
## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `make update-api` to update generated api files, if there's public API changes, otherwise the `verify-api-*` CI steps might fail.
 - [ ] Update [CHANGELOG.md](../CHANGELOG.md) or use the label 'skip changelog', otherwise `check changelog` CI step will fail.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

**Review by yunikkk — APPROVED** [review]

File: extension-style/src/main/java/com/mapbox/maps/extension/style/layers/generated/SkyLayer.kt [hunk 1]
```diff
@@ -917,7 +989,6 @@ class SkyLayer(override val layerId: String) : SkyLayerDsl, Layer() {
      *
      * This is an Expression representation of "sky-atmosphere-sun".
      *
-     * Position of the sun center [a azimuthal angle, p polar angle]. The azimuthal angle indicates the position of the sun relative to 0 degree north, where degrees proceed clockwise. The polar angle indicates the height of the sun, where 0 degree is directly above, at zenith, and 90 degree at the horizon. When this property is ommitted, the sun center is directly inherited from the light position.
```
Comment (pengdev) [inline extension-style/src/main/java/com/mapbox/maps/extension/style/layers/generated/SkyLayer.kt:920]: why are these sections being removed?

File: extension-style/src/main/java/com/mapbox/maps/extension/style/layers/generated/BackgroundLayer.kt [hunk 2]
```diff
@@ -32,6 +32,8 @@ class BackgroundLayer(override val layerId: String) : BackgroundLayerDsl, Layer(
   override val visibility: Visibility?
     /**
      * Whether this layer is displayed.
+     * 
+     * Use static method [defaultVisibility] to get the default property value.
```
Comment (pengdev) [inline extension-style/src/main/java/com/mapbox/maps/extension/style/layers/generated/BackgroundLayer.kt:36]:
```suggestion
     * Use static method [BackgroundLayer.defaultVisibility] to get the default property value.
```

File: extension-style/src/main/java/com/mapbox/maps/extension/style/layers/generated/SkyLayer.kt (hunk 1 above)
Comment (ank27) [reply extension-style/src/main/java/com/mapbox/maps/extension/style/layers/generated/SkyLayer.kt:920]: Duplicate doc entry.

File: extension-style/src/main/java/com/mapbox/maps/extension/style/layers/generated/BackgroundLayer.kt (hunk 2 above)
Comment (ank27) [reply extension-style/src/main/java/com/mapbox/maps/extension/style/layers/generated/BackgroundLayer.kt]: fixed that.

**Review by kiryldz — APPROVED** [review]

File: extension-style/src/main/java/com/mapbox/maps/extension/style/layers/generated/SkyLayer.kt (hunk 1 above)
Comment (pengdev) [reply extension-style/src/main/java/com/mapbox/maps/extension/style/layers/generated/SkyLayer.kt:920]: I don't think we should remove these sections, they are duplicated but also makes this API doc self-contained, user don't need to look for another function to see what it does.

**Review by pengdev — APPROVED** [review]

**Pull request merged**